config.m4

```diff
@@ -20,7 +20,6 @@ if test "$PHP_XDEBUG" != "no"; then
   CPPFLAGS="$INCLUDES $CPPFLAGS"
 
   AC_CHECK_FUNCS(gettimeofday)
-  AC_CHECK_HEADERS([sys/select.h])
```

Comment:
I understand that it can be confusing and hard to separate two threads talking about the same thing. When do you think that Xdebug 2.5 will be released? Do you have anything special you are wanting to put into that or are you going to keep supporting 2.4 at the same time as 2.5? It will be nice though to have IPv6 support in a "major" release :-)
Hopefully the code will compile for Windows and also work as expected.